Keas.Mvc/Controllers/FeedController.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Keas.Core.Data;
+using Keas.Mvc.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Keas.Core.Domain;
+
+namespace Keas.Mvc.Controllers
+{
+    public class FeedController : SuperController
+    {
+         private readonly ApplicationDbContext _context;
+
+        public FeedController(ApplicationDbContext context)
+        {
+            this._context = context;
+        }
+        public async Task<IActionResult> TeamFeed(Guid id, string space)
+        {            
+            if(space == "yes")
+            {
+                var people = await _context.People
+                .Where(x=> x.Team.Slug == Team && x.Active && x.Team.ApiCode == id)
```

Comment:
Instead of check the api code by adding it in here, you should do its own check first before going to the DB.  That way you can return the proper unauthorized status, instead of the 200 success status with no results they will get with a bad key now